pkg/testutil/setup.go

```diff
@@ -31,7 +31,7 @@ type SetupTestingEnvParams struct {
 	AdminSecretAccessKey string
 }
 
-func SetupTestingEnv(params *SetupTestingEnvParams) (logging.Logger, api.ClientWithResponsesInterface, *s3.S3) {
+func SetupTestingEnv(params *SetupTestingEnvParams) (logging.Logger, api.ClientWithResponsesInterface, *s3.S3, bool) {
```

Comment:
You don't need to return the value - you can just get it from viper after the setup

Reply:
I know, I thought it will make more sense to get it once
will change if you think it make more sense to get it from viper

Reply:
I think so, since you already have viper and it saves an unnecessary return parameter